packages/opentelemetry-exporter-datadog/test/datadogProbabilitySampler.test.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import * as api from '@opentelemetry/api';
+import {
+  DatadogProbabilitySampler,
+  DATADOG_ALWAYS_SAMPLER,
+  DATADOG_NEVER_SAMPLER,
+} from '../src';
+
+describe('DatadogProbabilitySampler', () => {
+  it('should return a always sampler for 1', () => {
```

Comment:
`an` not `a` and maybe put the name of class ?

Reply:
👍 , updated these test descriptions to be more clear +specify the class